policy/modules/admin/tomoyo-auditd.te

```diff
@@ -0,0 +1,32 @@
+policy_module(tomoyo-auditd, 1.0)
+
+########################################
+#
+# Declarations
+#
+
+type tomoyo_auditd_t;
+type tomoyo_auditd_exec_t;
+init_daemon_domain(tomoyo_auditd_t, tomoyo_auditd_exec_t)
+
+type tomoyo_auditd_etc_t;
+files_config_file(tomoyo_auditd_etc_t)
+
+type tomoyo_auditd_var_log_t;
+logging_log_file(tomoyo_auditd_var_log_t)
+
+########################################
+#
+# Tomoyo-auditd local policy
+#
+
+dev_rw_null(tomoyo_auditd_t)
+
+files_read_etc_files(tomoyo_auditd_t)
+read_files_pattern(tomoyo_auditd_t, tomoyo_auditd_etc_t, tomoyo_auditd_etc_t)
+
+logging_log_filetrans(tomoyo_auditd_t, tomoyo_auditd_var_log_t, file)
+manage_files_pattern(tomoyo_auditd_t, tomoyo_auditd_var_log_t, tomoyo_auditd_var_log_t)
+
+# Allow reading /sys/kernel/security/tomoyo
+selinux_get_enforce_mode(tomoyo_auditd_t)
```

Comment:
yes, this is a pickle, but i guess it could be worse

Reply:
I added a commit to declare the interface, so this hack shouldn't be needed anymore.